### PR TITLE
Moving to ai

### PR DESCRIPTION
**Pull request by MarkAYoder**
This is a quick hack to make things work.  It needs to be better thought out.

### CODE REVIEW

File: src/bone.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // pins is an object whose keys are the pinIndex keys, e.g. P9_14.
 // uarts and i2c are objects describing the serial ports and i2c buses.
 
-
+console.log("Loading bone.js");
```
Comment (jadonk) [inline src/bone.js]: I think this debug statement can be removed.

File: src/hw_mainline.js [hunk 2]
```diff
@@ -215,8 +215,10 @@ var exportGPIOControls = function (pin, direction, resp, callback) {
     if (debug) winston.debug('hw.exportGPIOControls(' + [pin.key, direction, resp] + ');');
     var n = isAI ? pin.ai.gpio : pin.gpio;
     var exists = my.file_existsSync(gpioFile[pin.key]);
+    console.log("exportGPIOControls, pin.key:", pin.key, gpioFile[pin.key]);
```
Comment (jadonk) [inline src/hw_mainline.js]: If you want to put this in, follow the if(debug) winston.debug pattern.

File: src/hw_mainline.js [hunk 3]
```diff
@@ -289,6 +291,8 @@ var enableAIN = function (callback) {
     return (ainPrefix);
 };
 
+// HACK to work on AI.
+const aimap = [0, 1, 3, 2, 7, 6, 4];  // Fixes AI's AIN mapping.
```
Comment (jadonk) [inline src/hw_mainline.js]: This needs to be removed now. I have the mapping as part of isAI.

File: src/index.js [hunk 4]
```diff
@@ -22,6 +22,8 @@ var rc = require('./rc');
 
 var debug = process.env.DEBUG ? true : false;
 
+console.log("DEBUG = " + debug);
```
Comment (jadonk) [inline src/index.js]: Have you not seen "DEBUG=true node script.js" work?

File: src/index.js [hunk 5]
```diff
@@ -158,7 +161,7 @@ f.pinMode = function (pin, direction, mux, pullup, slew, callback) {
                 } else
                     callback(err, false);
             }
-            return (false);
+            // return (false);		// Continue for now
```
Comment (jadonk) [inline src/index.js]: Didn't you find a fix in the code I added above this?  I don't think this is necessary.

File: src/index.js [hunk 6]
```diff
@@ -219,13 +221,16 @@ f.pinMode = function (pin, direction, mux, pullup, slew, callback) {
...
                     callback(resp.err, resp.value);
             }
-            return (resp.value);
+            // Move on for now.
+            // return (resp.value);
         }
     }
 
     // Enable GPIO and set direction
+    console.log('pinMode:  mux=', mux);
```
Comment (jadonk) [inline src/index.js]: Please clean these up if you want me to merge something.

File: src/index.js [hunk 7]
```diff
@@ -219,13 +221,16 @@ f.pinMode = function (pin, direction, mux, pullup, slew, callback) {
                 } else
                     callback(resp.err, resp.value);
             }
-            return (resp.value);
+            // Move on for now.
+            // return (resp.value);
```
Comment (jadonk) [inline src/index.js]: I don't think this is necessary anymore.

File: src/index.js [hunk 8]
```diff
@@ -810,4 +816,4 @@ f.setGlobals = function () {
     }
     global.run = run;
 };
-module.exports = f;
\ No newline at end of file
+module.exports = f;
```
Comment (jadonk) [inline src/index.js]: Always be sure to run 'npm run-script beautify' ahead of committing.

File: src/my.js [hunk 9]
```diff
@@ -382,6 +382,8 @@ var myeval = function (x) {
 };
 
 var getpin = function (pin) {
+    // console.log("getpin pin = " + pin);
```
Comment (jadonk) [inline src/my.js]: More comments in this file to clean up.

File: src/bone.js (hunk 1 above)
Comment (MarkAYoder) [reply src/bone.js]: Done

File: src/hw_mainline.js (hunk 3 above)
Comment (MarkAYoder) [reply src/hw_mainline.js]: Done.

File: src/index.js (hunk 5 above)
Comment (MarkAYoder) [reply src/index.js]: Yup, it works now.

File: src/index.js (hunk 6 above)
Comment (MarkAYoder) [reply src/index.js]: Cleaned up

File: src/index.js (hunk 7 above)
Comment (MarkAYoder) [reply src/index.js]: Fixed

**Comment by jadonk** [comment]
I believe any additional function added by this pull request has already been added. Please re-open if I missed something.

**Pull request closed without merging**